sphinx/api/configuration/suite.rst

```diff
@@ -34,7 +34,7 @@ A suite directory may contain the following:
 
 .. rose:file:: rose-suite.conf
 
-   The suite install configuration file :rose:file:`rose-suite.conf` should
+   The suite install configuration file ``rose-suite.conf`` should
```

Comment:
I know that we are in the `rose-suite.conf` file so its a little un-necessary to reference the file but, literal strings appear differently [in orange] to referenced objects [in black] in the read_the_docs theme (and likely in others too) so for consistency of presentation I've been using the rose directive for all references.

I appreciate that there is a little inconsistency in this as content has been moved from the old documentation, we'll just have to fix them as we go.

Reply:
Okay no problem - I actually noticed this in the case of the following section, where `rose-suite.info` is just a literal string (quoted in backticks), so I was just trying to make those consistent. I will revert this and change `rose-suite.info` into a domain for consistency.